satviz-consumer/src/main/java/edu/kit/satviz/consumer/config/WeightFactor.java

```diff
@@ -0,0 +1,52 @@
+package edu.kit.satviz.consumer.config;
+
+import java.util.function.DoubleUnaryOperator;
+
+/**
+ * This enum contains the possible weight factors, by which each clause
+ * affects the corresponding edge weights in the graph.
+ *
+ * @author johnnyjayjay
+ */
+public enum WeightFactor {
```

Comment:
In den Beschreibungen der Strategien steht "adds to the edge weight", dabei beschreiben die Konstanten **Faktoren** für Kantenänderungen.

Reply:
Javadoc ist jetzt angepasst!